cv_next/app/profile/[profileId]/components/profilePersonalData.tsx

```diff
@@ -0,0 +1,84 @@
+"use client";
+
+import Image from "next/image";
+import { ImagePlusIcon, PencilIcon } from "lucide-react";
+import { useEffect, useState } from "react";
+import DynamicProfileImage from "@/components/ui/DynamicProfileImage";
+import { useUser } from "@/hooks/useUser";
+import ProfileForm from "./profileForm";
+import { ProfileDisplay } from "./profileDisplay";
+
+export const ProfilePersonalData = ({
+  user,
+  revalidationFn,
+}: {
+  user: UserModel;
+  revalidationFn: () => void;
+}) => {
+  const [isEditing, setIsEditing] = useState(false);
+  const { userData } = useUser();
+  const [isCurrentUser, setIsCurrentUser] = useState(false);
+
+  useEffect(() => {
+    if (userData && user.id === userData.id) {
+      setIsCurrentUser(true);
+    } else {
+      setIsCurrentUser(false);
+    }
+  }, [user.id, userData]);
+
+  const changePfp = () => {
+    //To be implemented
+  };
```

Comment:
Maybe there should be like a "Coming soon" thing when the user clicks on the option itself?

Reply:
I do think we need to handle it as part of V1, no?
If not, then sure. lmk.

Reply:
tbf if we don't have it as part of V1 I can simply remove this edit option.

Reply:
It's just that currently the way we handle the images is just via the links from gmail(we don't upload any image or something to our own bucket)

Reply:
So, we currently don't have the means to edit the image, correct?
If not, I'll remove the edit option, and we can add it as an issue to V2.

Reply:
Exactly, we don't
I'll add it to V2